### PR TITLE
Upload files on drag-and-drop to the textarea

### LINKED ISSUE #1188: Feature request - drag and drop file to HTTP Upload
I dragged and dropped a file into inverse.chat - Firefox opened the file instead, and when I pressed back I was logged out :)

Firefox 62.0 (32-bit), Debian GNU/Linux 9.5 (stretch)

### PR DESCRIPTION
**Pull request by linkmauve**
When one or more files are dropped on the textarea, they get uploaded or an error is printed if this isn’t possible.

There is also a fix to correctly displays an error when XEP-0363 isn’t supported by the server.

Fixes #1188.

### CODE REVIEW

File: dist/converse.js [hunk 1]
```diff
@@ -59128,6 +59130,19 @@ _converse_headless_converse_core__WEBPACK_IMPORTED_MODULE_5__["default"].plugins
...
       },
 
+      onDragOver(evt) {
+        evt.preventDefault();
+      },
+
+      onDrop(evt) {
+        /* There are no files to be dropped, this isn’t a file transfer
+         * operation.
+         */
```
Comment (jcbrand) [inline dist/converse.js]: This comment confuses me. Generally I use the `/* */` form as docstrings explaining the purpose of the method/function.

**Comment by jcbrand** [comment]
Thanks! Glad to have this feature.

File: dist/converse.js (hunk 1 above)
Comment (linkmauve) [reply dist/converse.js]: Oh, usually I see `/** … */` in the role of docstrings; do you prefer `//` for plain comments then?

Comment (linkmauve) [reply dist/converse.js]: This is now fixed in 9a12afc70230a45afed180756ff1c8d24a0edb37.

**Pull request merged**